docs/source/config_update.md

```diff
@@ -966,15 +966,15 @@ configuration of the channel being updated (with the exception of the ordering
 system channel, whose configuration is copied into the configuration of
 application channels by default).
 
-Note: this topic presumes that you are operating from within a peer container.
-
-* ``CH_NAME``: the name of the channel being updated.
-* ``ORDERER_CA``: the absolute path to the root CA certificate of the organization
-  proposing the system channel update.
+* `CH_NAME`: the name of the channel being updated.
+* `TLS_ROOT_CA`: the path to the root CA cert of the TLS CA of the organization
+  proposing the update.
```

Comment:
In all the upgrade docs, TLS_ROOT_CA is only used to connect to the ordering node to fetch/update config. Therefore, it is `the absolute path to the TLS cert of your ordering node(s)`.